localization/english/wc_game_concepts_l_english.yml

```diff
@@ -35,4 +35,14 @@
  
  game_concept_grim_religion:0 "Grim Religion"
  game_concept_grim_religions:0 "Grim Religions"
- game_concept_grim_religion_desc:0 "A [religion|E] so dark and evil that it opposes the rest of the world. They don't seek outside allies and won't stop until they reach their ultimate goal.\n\nThese religions are:\n$EFFECT_LIST_BULLET$$deathbringer$\n$EFFECT_LIST_BULLET$$doombringer$\n$EFFECT_LIST_BULLET$$shathgral$"
\ No newline at end of file
+ game_concept_grim_religion_desc:0 "A [religion|E] so dark and evil that it opposes the rest of the world. They don't seek outside allies and won't stop until they reach their ultimate goal.\n\nThese religions are:\n$EFFECT_LIST_BULLET$$deathbringer$\n$EFFECT_LIST_BULLET$$doombringer$\n$EFFECT_LIST_BULLET$$shathgral$"
+
+ game_concept_stunning:0 "Stunning"
+ game_concept_wc_stunned:0 "Stunned"
+ game_concept_stunned:0 "Stunned"
+ game_concept_stun:0 "Stun"
+ game_concept_wc_stunned_desc:0 "A stunned character is unable to interract with any other character. Character interactions include, but are not limited to, declaring war, calling allies to war, using hooks, changing vassal contracts, and more."
```

Comment:
Reminder. War, allies, hooks, vassal contacts are game concepts too and should be linked here as well instead of plain text.